framework/db/SchemaBuilderTrait.php

```diff
@@ -37,62 +35,245 @@
 trait SchemaBuilderTrait
 {
     /**
-     * @var Connection|array|string the DB connection object or the application component ID of the DB connection
+     * @return Connection the database connection to be used for schema building.
+     */
+    protected abstract function getDb();
+
+    /**
+     * Creates a primary key column.
+     * @param integer $length column size or precision definition.
+     * This parameter will be ignored if not supported by the DBMS.
+     * @return ColumnSchemaBuilder the column instance which can be further customized.
+     * @since 2.0.6
```

Comment:
did this because it is a trait and the method will show up in the Migration class. This way it is clear which are new from the docs.